### ISSUE TITLE
Graphing mode: Equation clear button cannot be accessed via keyboard

### ISSUE THREAD

**Issue by joshkoon** [microsoft/calculator#951]
**Describe the bug**
The clear button that appears on the equation edit box is not accessible via keyboard or with Narrator. 

**Steps To Reproduce**

1. Go to Graphing mode
2. Add an equation, such as "x + 2"
3. Observe that while editing the equation, a Clear (X) button appears in the edit box.
4. Attempt to navigate to the button via the Tab key on keyboard, or via screen reader shortcuts with Narrator enabled.

**Expected behavior**
Clear button should be available to users navigating via the keyboard.

**Screenshots**

**Device and Application Information**
- OS Build: 10.0.19547.0
- Architecture: x64
- Application Version: 10.1912.4.0
- Region: en-US
- Dev Version Installed:

**Additional context**

**Requested Assignment**

I'm just reporting this problem.  I don't want to fix it.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I've seen this issue come in and have gone to tell a human about it.

**Comment by grochocki** [comment]
Thanks for the detailed report! This behavior is consistent with the default behavior of TextBox controls, so I am going to close this as by design.
